src/constants.js

```diff
@@ -12,6 +12,12 @@ export const TAWASOL_BASE_PATH_API =
   'https://madaportal.org/tawasol/wp-json/custom/v2/dea44ade76/';
 export const TAWASOL_BASE_IMAGE_URL =
   'https://www.madaportal.org/tawasol/wp-content/uploads/images/';
+export const AZURE_VOICES_BASE_PATH_API =
+  'https://eastus.tts.speech.microsoft.com/cognitiveservices/voices/';
```

Comment:
Are we only getting American voices if we list voices for the East US region? Should we be using `AZURE_SPEECH_SERVICE_REGION` here?